flows/yosys/synth.tcl

```diff
@@ -0,0 +1,30 @@
+# Default Yosys synthesis tcl script for the yosys_synth_file_step rule.
+# It can be replaced by a user-defined script by overriding the synth_tcl
+# argument of that rule.
+
+yosys -import
+
+# read design
+set input_rtl $::env(INPUT_RTL)
+yosys read_verilog -sv -defer $input_rtl
```

Comment:
Are we currently limited to a single verilog file in this mode?

Reply:
Yes. yosys_synth_library_step will be a different rule (because it will need to juggle a filelist). That annoys me a bit, but I comfort myself with knowing that we don't need to keep the parallel paths downstream from here.